modules/Bio/EnsEMBL/Variation/Pipeline/RegulationEffect_conf.pm

```diff
@@ -88,8 +88,12 @@ sub resource_classes {
     # requirements, queue parameters etc.) to suit your own data
 
     return {
-      'default' => { 'LSF' => '-qproduction -R"select[mem>2000] rusage[mem=2000]" -M2000'},
-      'highmem' => { 'LSF' => '-qproduction -R"select[mem>15000] rusage[mem=15000]" -M15000'},
+      'default' => { 'LSF' => '-qproduction -R"select[mem>2000] rusage[mem=2000]" -M2000',
+                     'SLURM' => "--partition=production --time=4:00:00 --mem=2G" },
+      'highmem' => { 'LSF' => '-qproduction -R"select[mem>15000] rusage[mem=15000]" -M15000',
+                     'SLURM' => "--partition=production --time=28:00:00 --mem=15G"},
```

Comment:
Does `highmem` also need a long runtime?

Reply:
Not for now, 28h is probably enough for all jobs.